File: src/lib/copy/__test__/examples/RestConstants.js
/* eslint-disable */
// path params
export const FIZZ_PARAM = 'fizz';
export const BAZ_PARAM='baz';

// paths
export const FOO = "/foo";
export const FOO_BAR = `/foo/bar`;
export const FOO_101_BAR = '/foo/101/bar'
// export const COMMENTED_CONSTANT = "/commented/constant";
/**
 * export const COMMENTED_CONSTANT = "/commented/constant";
 */
export const FOO_FIZZ = FOO + `/{${ FIZZ_PARAM }}`;
export const POW = `/pow/${{BAZ_PARAM}}`

export const FOO_FIZZ2 = FOO + '/{' + FIZZ_PARAM + '}';
export const POW2 = '/pow/{' + BAZ_PARAM  + "}"

File: src/lib/copy/__test__/examples/RestConstants.java
public class RestConstants {
    // path params
    public static final String FIZZ_PARAM = "fizz";
    public static final String BAZ_PARAM="baz";

    // paths
    public static String FOO = "/foo";
    static final String FOO_BAR = "/foo/bar";
    static String FOO_101_BAR = "/foo/101/bar";
    // public static final String COMMENTED_CONSTANT = "/commented/constant";
    /**
    * export const COMMENTED_CONSTANT = "/commented/constant";
    */
    public static final String FOO_FIZZ = FOO + "/{" + FIZZ_PARAM + "}";
    public static final String POW = "/pow/"+BAZ_PARAM;
}
